pysatNASA/instruments/templates/template_cdaweb_instrument.py

```diff
@@ -1,8 +1,18 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
 """Template for a pysat.Instrument support file that utilizes CDAWeb methods.
 
 Copy and modify this file as needed when adding a new Instrument to pysat.
 
+DO NOT include the NRL distribution statement in your new file.
```

Comment:
```suggestion
DO NOT include the NRL distribution statement in your new file. This will have to be
approved by NRL separately before the versioned release.
```
@aburrell, does this work?